Add 0.0.0.0 to localhost Filter

diarmuidie opened the pull request:
Some development environments will bind to the local IP 0.0.0.0 (Jekyll run in Docker on Mac is one example). The tracking JS code should treat these as localhost and filter them out so that they don't show up in the tracking dashboard:

<img width="665" alt="Screenshot 2021-05-09 at 09 32 06" src="https://user-images.githubusercontent.com/707037/117565490-81e28600-b0a9-11eb-895b-f49100a08791.png">

arp242 commented:
Seems like a good change :+1: 

You can't modify the count.v1.js and count.v2.js though; only count.js. The v*.js are [stable versions intended to be used with SRI](https://www.goatcounter.com/code/countjs-versions); changing it would break it for people as the hash no longer matches.

Bonus points if you also update the little changelog on that page btw, but I can do it afterwards as well. It's in `tpl/code/countjs-versions.markdown`.

diarmuidie commented:
Ah, I didn't realise those were stable version. Makes sense!

I've removed the change from the stable versions and added a line to the changelog https://github.com/zgoat/goatcounter/pull/483/commits/df66c46c95ff23aea5726250c2994d18dfb8c7cd

arp242 commented:
Cheers, thanks!
